coral/src/app/router.tsx

```diff
@@ -60,13 +62,18 @@ import {
   Routes,
   TOPIC_OVERVIEW_TAB_ID_INTO_PATH,
   TopicOverviewTabEnum,
-} from "src/app/router_utils";
-import { getRouterBasename } from "src/config";
+} from "src/services/router-utils/types";
 import {
   createPrivateRoute,
   createRouteBehindFeatureFlag,
-} from "src/services/feature-flags/route-utils";
-import { FeatureFlag } from "src/services/feature-flags/types";
+  filteredRoutesForSuperAdmin,
+  SuperadminRouteMap,
+} from "src/services/router-utils/route-utils";
+import { isFeatureFlagActive } from "src/services/feature-flags/utils";
+
+const superAdminAccessCoralEnabled = isFeatureFlagActive(
```

Comment:
We don't need to run the filter when the feature isn't enabled anyway, so I added it here, too.